public/javascripts/application.js

```diff
@@ -1,19 +1,52 @@
-$(document).on('click', '#secret_submit', function () {
-  $.post('/encrypt/get_ct_json', {
-    plaintext_secret: $('#plaintext_secret').val()
-  },
-  function(json) {
-    if (json.error) {
-      $('#alert').addClass('alert-error');
-      $('#alert').attr('style', 'display: block !important');
-      if (json.error.message) {
-        $('#alert').html(json.error.message);
+'use strict';
+
+document.addEventListener('DOMContentLoaded', function() {
+  var submit = document.querySelector('#secret_submit');
+  var alert = document.querySelector('#alert');
+  var response = document.querySelector('#response');
+
+  submit.addEventListener('click', function() {
+    var request = new XMLHttpRequest();
+    var data = {
+      plaintext_secret: document.querySelector('#plaintext_secret').value
```

Comment:
I can! See https://github.com/rapid7/acs/blob/deprecate-jquery/views/index.ejs#L22 although this will get refactored later.